layouts/auto_generated_product.tpl

```diff
@@ -106,7 +108,16 @@
                     <div class="buy-btn-content js-buy-btn-content">
                       {% include "buy-button" %}
                     </div>
-                    {% content bind=product %}
+
+                    <div class="product-cross-page-info">
```

Comment:
This div should not be displayed when there has no content:
E.g. `{% if editmode or gallery_content_size > 0 %}`